.freight.non-experimental.yml

```diff
@@ -0,0 +1,56 @@
+sentry:
+  organization: sentry
+  project: snuba
+  repository: getsentry/snuba
+
+steps:
+  - kind: KubernetesDeployment
+    selector:
+      label_selector: service=snuba
+    containers:
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: api
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: errors-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: errors-replacer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: errors-tiger-replacer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: replacer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: transactions-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: sessions-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: outcomes-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: loadbalancer-outcomes-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: outcomes-billing-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: events-subscriptions-scheduler
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: events-subscriptions-executor
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: transactions-subscriptions-scheduler
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: transactions-subscriptions-executor
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: cdc-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: cdc-groupassignee-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: querylog-consumer
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: snuba-admin
+  - kind: KubernetesCronJob
+    selector:
+      label_selector: service=snuba
+    containers:
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: cleanup
+      - image: us.gcr.io/sentryio/snuba:{sha}
+        name: optimize
```

Comment:
I can remove this but for the sake of keeping this file as the standard for a minimum production ready config I've left it in

Reply:
I think you have the right idea. This should be everything except experimental consumers.